web/src/components/storage/LvmPage.tsx

```diff
@@ -0,0 +1,130 @@
+  Label,
+  TextInput,
+} from "@patternfly/react-core";
+import { Page, SubtleContent } from "~/components/core";
+import { useAvailableDevices } from "~/queries/storage";
+import { deviceLabel } from "./utils";
+import { contentDescription, filesystemLabels, typeDescription } from "./utils/device";
+import { _ } from "~/i18n";
+
+/**
+ * Form for creating a LVM volume group
+ */
+export default function LvmPage() {
+  const allDevices = useAvailableDevices();
+  const [name, setName] = useState("system");
+  // FIXME: decide what to store, if the device object or just its sid and type
+  // the state accordingly
+  const [selectedDevices, setSelectedDevices] = useState([]);
+  const [moveMountPoints, setMoveMountPoints] = useState(true);
+
+  const updateName = (_, value) => setName(value);
+  const updateSelectedDevices = (value) => {
+    setSelectedDevices(
+      selectedDevices.includes(value)
+        ? selectedDevices.filter((d) => d !== value)
+        : [...selectedDevices, value],
+    );
+  };
+
+  const onSubmit = () => {
+    console.log("TODO: implement the logic to be triggered when LVM form is submitted");
+  };
+
+  return (
+    <Page>
+      <Page.Header>
+        <Content component="h2">{_("New Volume Group")}</Content>
+        <SubtleContent>{_("Create a new LVM volume group")}</SubtleContent>
+      </Page.Header>
+
+      <Page.Content>
+        <Form id="lvmForm" onSubmit={onSubmit}>
+          <FormGroup fieldId="lvmName" label={_("Name")} isStack>
+            <TextInput id="lvmName" value={name} onChange={updateName} />
+          </FormGroup>
+          <FormGroup label={_("Disks")} role="group" style={{ justifySelf: "stretch" }} isStack>
+            <Content component="small">
+              {_(
+                "The needed LVM physical volumes will be created as partitions on the chosen disks, based on the sizes of the logical volumes. If you select more than one disk, the physical volumes may be distributed along several disks.",
+              )}
+            </Content>
+            <Gallery hasGutter>
+              {allDevices.map((device) => (
+                <Checkbox
+                  key={device.sid}
+                  id={`device${device.sid}`}
+                  label={<Content isEditorial>{deviceLabel(device)}</Content>}
+                  description={
+                    <Flex rowGap={{ default: "rowGapXs" }} columnGap={{ default: "columnGapSm" }}>
+                      <SubtleContent>
+                        {typeDescription(device)} {contentDescription(device)}
+                      </SubtleContent>
+                      {filesystemLabels(device).map((s, i) => (
+                        <Label key={i} variant="outline" isCompact>
+                          {s}
+                        </Label>
+                      ))}
+                      {device.systems.map((s, i) => (
+                        <Label key={i} isCompact>
+                          {s}
+                        </Label>
```

Comment:
Time to create a dummy component for consistently rendering both, file systems labels and systems names instead of duplicating these <Labels> whenever they are needed.